wire-ios-request-strategy/Sources/Request Strategies/User Clients/FetchingClientRequestStrategy.swift

```diff
@@ -389,6 +395,22 @@ final class UserClientByQualifiedUserIDTranscoder: IdentifierObjectSyncTranscode
         }
     }
 
+    private func markAllClientsAsUpdated(identifiers: Set<QualifiedID>) {
+        for identifier in identifiers {
+            if let user = ZMUser.fetch(
+                with: identifier.uuid,
+                domain: identifier.domain,
+                in: managedObjectContext
+            ) {
+                for client in user.clients {
+                    client.needsToBeUpdatedFromBackend = false
```

Comment:
to fully understand, I guess this property has been set to true before making the request, right?

Reply:
yes that's correct